Userland/Libraries/LibWeb/Loader/FrameLoader.cpp

```diff
@@ -187,6 +187,11 @@ bool FrameLoader::load(LoadRequest& request, Type type)
     if (type == Type::IFrame)
         return true;
 
+    auto document = browsing_context().active_document();
+    if (document && document->has_active_favicon()) {
+        return true;
+    }
```

Comment:
Since `active_document()` returns a pointer it is nice to use `auto* document` to indicate there is no copy here.

Also we generally don't use brackets for single line blocks

Also holds for the two copies of this below here